content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -6,10 +6,21 @@
 
 # Prepare Your Repositories
 
-Iterative Studio creates projects by identifying datasets, metrics and
-hyperparameters defined in your Git repositories. These values are stored in
-your Git repositories as JSON or YAML files. You can add these values to your
-Git repositories in two ways:
+Iterative Studio creates [projects] by identifying machine learning models,
+datasets, metrics and hyperparameters defined in your Git repositories. These
+values are found in JSON or YAML files in the repository. Additionally, model
+information may be available as Git tags.
```

Comment:
Follow up to https://github.com/iterative/dvc.org/pull/3778#discussion_r926428781

```suggestion
Iterative Studio creates [projects] by identifying machine learning models,
datasets, metrics, and hyperparameters defined in your Git repositories. These
values are mainly found in JSON or YAML files in the repository. Additional
model information may also be available in Git tags.
```